src/utils/config.json

```diff
@@ -25,7 +25,16 @@
         "name": "Treatments",
         "component": "TR",
         "displayOrder": 5
+      },
+      {
+        "name": "Nursing Tasks",
+        "component": "NT",
+        "displayOrder": 6
       }
-    ]
+    ],
+    "nursingTasks": {
+      "timeInMinutesFromNowToShowTaskAsRelevant": 60,
```

Comment:
Do we keep it here or keep it as part of a config. where is this getting used?

Reply:
all the configs for this repo as per previous discussion , for now would reside within config.json